WordPress/src/main/res/layout/restore_progress_fragment.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
```

Comment:
Minor (🔍): This comment does not relate to this layout file but its corresponding `Backup` layout file, where there is an unused `xmlns:tools` namespace. Consider cleaning this up.

Reply:
Noted - there will only be 1 fragment xml in the future - they are all the same.